test/test_jit.py

```diff
@@ -9290,6 +9290,17 @@ def fn(lst):
 
         self.checkScript(fn, ([torch.ones(2) + 2, torch.ones(2)],))
 
+    def test_view_write(self):
+        def fn(x, y):
+            l = []
+            l.append(x)
+            x_view = l[0]
+            a = x + x
+            x_view.add_(y)
+            b = x + x
+            return a == b
+        self.checkScript(fn, (torch.rand(2, 3), torch.rand(2, 3)))
```

Comment:
This is testing that a and b do not get CSEd right ?

Reply:
yeah

Reply:
This test is somewhat flaky, as it depends on us being really bad at inferring alias sets of tensors indexed out of lists, but I guess it's the best we can do right now 🤷‍♂️

Reply:
yeah—when we improve aliasing information it'll stop testing this specific situation. Although it's a useful case to have under test anyway.

Reply:
I mentioned in a different comment - would be safer to test with a python op than list index, since those will always wildcard